src/backend/api/Fusion.Resources.Domain/Commands/Requests/AddComment.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Fusion.Resources.Database;
+using Fusion.Resources.Database.Entities;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Domain.Commands
+{
+    public class AddComment : TrackableRequest
+    {
+        public AddComment(Guid requestId, string comment, string origin)
+        {
+            RequestId = requestId;
+            Comment = comment;
+            Origin = origin;
+        }
+
+        public Guid RequestId { get; }
+
+        public string Comment { get; }
+
+        public string Origin { get; }
+
+        public Guid CreatedById { get; set; }
```

Comment:
Is this needed when using trackable request and editor?

Reply:
No. Started out this way and ended up using Editor, removing.